fastlane/lib/fastlane/actions/git_commit.rb

```diff
@@ -10,8 +10,17 @@ def self.run(params)
 
         skip_git_hooks = params[:skip_git_hooks] ? '--no-verify' : ''
 
-        result = Actions.sh("git commit -m #{params[:message].shellescape} #{paths} #{skip_git_hooks}".strip)
-        UI.success("Successfully committed \"#{params[:path]}\" 💾.")
+        command = "git commit -m #{params[:message].shellescape} #{paths} #{skip_git_hooks}".strip
+        nothing_to_commit = false
```

Comment:
I don't think we need this. The if statement is early returning out of the method don't need this 😊